admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/toolingpage/toolingpage.html

```diff
@@ -111,80 +111,89 @@
-        $perAdminApp.loadComponent('admin-components-iconopenfolder')
-        $perAdminApp.loadComponent('admin-components-iconrename')
-        $perAdminApp.loadComponent('admin-components-draghandle')
-        $perAdminApp.loadComponent('admin-components-iconbrowser')
-        $perAdminApp.loadComponent('pagerendervue-components-placeholder')
-
+        [
+            'admin-components-iconaction',
+            'admin-components-action',
+            'admin-components-notifyuser',
+            'admin-components-askuser',
+            'admin-components-promptuser',
+            'admin-components-pathbrowser',
+            'admin-components-logo',
+            'admin-components-extensions',
+            'admin-components-spinner',
+            'admin-components-tour',
+            'admin-components-about',
+            'admin-components-explorerpreview',
+            'admin-components-explorerpreviewnavitem',
+            'admin-components-explorerpreviewcontent',
+            'admin-components-confirmdialog',
+            'admin-components-fileupload',
+            'admin-components-separator',
+            'admin-components-colorpaletteselector',
+            'admin-components-materializeswitch',
+            'admin-components-componentexplorer',
+            'admin-components-nodetreeitem',
+            'admin-components-publishingmodal',
+            'admin-components-publishinginfo',
+            'admin-components-richtoolbar',
+            'admin-components-richtoolbargroup',
+            'admin-components-richtoolbarbtn',
+            'admin-components-addcomponentmodal',
+            'admin-components-icon',
+            'admin-components-tabswrapper',
+            'admin-components-tab',
+
+            'admin-components-materializedropdown',
+            'admin-components-dropdowndivider',
+            'admin-components-materializemodal',
+            'admin-components-materializespinner',
+
+            'field-pathbrowser',
+            'field-texteditor',
+            'field-list',
+            'field-collection',
+            'field-materialswitch',
+            'field-material-checkbox',
+            'field-material-radios',
+            'field-material-select',
+            'field-material-checklist',
+            'field-material-datepicker',
+            'field-material-timepicker',
+            'field-material-datetime',
+            'field-material-textarea',
+            'field-material-multiselect',
+            'field-material-range',
+            'field-listselection',
+            'field-iconbrowser',
+            'field-headline',
+            'field-horizontal-line',
+
+            'admin-components-iconeditpage',
+            'admin-components-iconopenfolder',
+            'admin-components-iconrename',
+            'admin-components-draghandle',
+            'admin-components-iconbrowser',
+            'pagerendervue-components-placeholder'
+        ].forEach(name => $perAdminApp.loadComponent(name))
```

Comment:
we are not transpiling this file. will this change break backwards compatibility?

Reply:
https://caniuse.com/?search=foreach
I think we are good

Reply:
not on the => part: https://caniuse.com/?search=%3D%3E

Reply:
Okay we could either swap it with `function() {}` then or (as suggested by you in slack) refer to a function that is written inside a file that is transpiled

Reply:
`function() {}` applied